internal/pkg/service/cli/dialog/create_table.go

```diff
@@ -101,3 +170,46 @@ func getOptionCreateRequest(columns []string) []keboola.Column {
 
 	return c
 }
+
+func parseColumnsDefinitionFromFile(input string) ([]keboola.Column, error) {
+	var result []keboola.Column
+
+	err := yaml.Unmarshal([]byte(input), &result)
+	if err != nil {
+		return nil, err
+	}
+	return result, err
+}
+
+func (p *Dialogs) defaultValue() string {
+	fileHeader := `#Command "remote create table"
+
+#The CLI command creates a new storage table, and the table can be defined either by specifying the full table ID in the argument or by using the --bucket and --name flags.
+#In addition, you can specify column names and specify the primary key using the appropriate flags.
+
+#Using the --columns-from flag to specify a yaml definition file with columns types.
+
+#Another way to define columns is to use the --columns-from flag and then specify the path to your json file.
+
+
+#Example definition file for --columns-from flag:
+
```

Comment:
This help should be modified: there is no point in mentioning flags in interactive mode, right?

Reply:
Imagine that you are a user, the editor opens, and help, in the comments, says you what to do.